src/frontend/next/src/components/ExistingFeedList.tsx

```diff
@@ -0,0 +1,70 @@
+import { Grid, TextField, Typography } from '@material-ui/core';
+import { AccountCircle, RssFeed } from '@material-ui/icons';
+import { FeedHash } from '../interfaces';
+import DeleteFeedDialogButton from './DeleteFeedDialogButton';
+
+type ExistingFeedListProps = {
+  feedHash: FeedHash;
+  deletionCallback: (id: string) => void;
+};
+
+function ExistingFeedList({ feedHash, deletionCallback }: ExistingFeedListProps) {
```

Comment:
const ExistingFeedList = ({ feedHash, deletionCallback }: ExistingFeedListProps) => {...}

Let's use ES6.

Reply:
Noted and fixed